src/breadcrumbs/_base.scss

```diff
@@ -24,13 +24,18 @@
 
 .breadcrumbs-list-item {
   display: inline-block;
-  margin: 0 ms(-7);
+  margin-right: ms(-7);
 
-  &:first-child {
-    margin-left: 0;
+  &:after {
+    content: '>';
```

Comment:
@misscs What's the translation benefit of using the unicode character over the character itself?
@robwierzbowski was wondering too